extensions/voice/org.eclipse.smarthome.voice.mactts/src/main/java/org/eclipse/smarthome/voice/mactts/internal/MacTTSAudioStream.java

```diff
@@ -74,19 +79,33 @@ private InputStream createInputStream() throws AudioException {
         try {
             Process process = Runtime.getRuntime().exec(command);
             process.waitFor();
-            File file = new File(outputFile);
-            if (file.exists()) {
-                return new FileInputStream(file);
-            } else {
-                throw new AudioException("Temporary file '" + outputFile + "' not found!");
+            file = new File(outputFile);
+            if (file != null) {
```

Comment:
`new File(...)` should never return null but always a reference (if no exception is thrown by the construction).

If the file does not exist 0L is returned, so I think we do not need any check at all.